lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1888,5 +1888,5 @@ def _get_url(self, course_key, checkpoint):
                        kwargs={
                            "course_id": unicode(course_key),
                            "checkpoint_name": checkpoint,
-                           "location": unicode(self.reverification_location)
+                           "usage_id": unicode(self.reverification_location)
```

Comment:
Instead of defining the new regex he used the existing variable.
https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L565
That’s why he changed the variable name.